### DISCUSSION TITLE
Why hardcoded attributes like rounded?

### DISCUSSION THREAD

**Discussion by ekandreas** [filamentphp/filament#626] [General]
In the form components the class definitions to tailwind is harcoded for some important design results.
Eg, the text form has rounded-xl. Why? How to overwrite that ugly form field attribute?

**Comment by roni-estein** [comment]
You can publish one or multiple views to your own app.

1. Make sure you have a git commit before you run this or you can't cherry pick.
2. `php artisan vendor:publish --tag="forms-views"`
3. Pick the items you want to overwrite, and change them to your hearts content.
4. Commit the updated component views ( I change ONLY what I need to, to make it easier on myself during an upgrade )
5. `git clean -df` => the infamous  `nah`, This will purge everything that is not committed, even your untracked files.
  5.1 composer install
  5.2 npm install && npm run dev
6. For a safer `nah` experience with backups you can also refer to this gist ( by me, after a disastrous nah ) https://gist.github.com/roni-estein/190c52f9c03c6ad6a883c9d0f0320c4a

You will now have a repo in your prior state with only the files left to change the components you desired!
- Cheers
